rules/linux/privilege_escalation_load_and_unload_of_kernel_via_kexec.toml

```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2023/06/09"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/09"
+integration = ["endpoint"]
+
+[rule]
+author = ["Elastic"]
+description = """
+This detection rule identifies the usage of kexec, helping to uncover unauthorized kernel replacements and potential 
+compromise of the system's integrity. Kexec is a Linux feature that enables the loading and execution of a different 
+kernel without going through the typical boot process. Malicious actors can abuse kexec to bypass security measures, 
+escalate privileges, establish persistence or hide their activities by loading a malicious kernel, enabling them to 
+tamper with the system's trusted state, allowing e.g. a VM Escape. 
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Kernel Load or Unload via Kexec Detected"
+references = [
+    "https://www.crowdstrike.com/blog/venom-vulnerability-details/",
+    "https://www.makeuseof.com/what-is-venom-vulnerability/",
+    "https://madaidans-insecurities.github.io/guides/linux-hardening.html"
+]
+risk_score = 73
+rule_id = "4d4c35f4-414e-4d0c-bb7e-6db7c80a6957"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Persistence", "Privilege Escalation", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "linux" and event.action == "exec" and process.name == "kexec" and 
+process.args in ("--exec", "-e", "--load", "-l", "--unload", "-u")
```

Comment:
I could be wrong, but I think having the `--exec` and `-e` options could also flag on kexec calls where different kernel modules were not loaded or unloaded. Is this intentional? Fine if it is, just wanting to double check. [Reference](https://linux.die.net/man/8/kexec).

Reply:
This was intentional, as the --exec is eventually used to actually execute the loaded kernel; which I figured would also be a good flag to add. This is very suspicious behavior in general, unless the person using kexec is a linux developer. Good to point it out though, because it indeed isn't fully reflected in the rule name.